From 6047f0c7bc371f229c33bb4d7cefa1b2377c77c1 Mon Sep 17 00:00:00 2001
From: Hugh McMaster <hugh.mcmaster@outlook.com>
Date: Thu, 29 Oct 2015 23:17:20 +1100
Subject: kernel32: Implement GetNumberOfConsoleFonts (resend)

This function is called by GetConsoleFontSize and its tests.

Note that we only ever have one entry in the console font table.

Signed-off-by: Hugh McMaster <hugh.mcmaster@outlook.com>
---
 dlls/kernel32/console.c     | 5 +++++
 dlls/kernel32/kernel32.spec | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/dlls/kernel32/console.c b/dlls/kernel32/console.c
index c32dc49..67627cc 100644
--- a/dlls/kernel32/console.c
+++ b/dlls/kernel32/console.c
@@ -3234,6 +3234,11 @@ BOOL WINAPI SetConsoleIcon(HICON icon)
     return FALSE;
 }
 
+DWORD WINAPI GetNumberOfConsoleFonts(void)
+{
+    return 1;
+}
+
 BOOL WINAPI GetCurrentConsoleFont(HANDLE hConsole, BOOL maxwindow, LPCONSOLE_FONT_INFO fontinfo)
 {
     BOOL ret;
diff --git a/dlls/kernel32/kernel32.spec b/dlls/kernel32/kernel32.spec
index d7eab83..9b2a131 100644
--- a/dlls/kernel32/kernel32.spec
+++ b/dlls/kernel32/kernel32.spec
@@ -753,7 +753,7 @@
 @ stdcall GetNumberFormatA(long long str ptr ptr long)
 # @ stub GetNumberFormatEx
 @ stdcall GetNumberFormatW(long long wstr ptr ptr long)
-@ stub GetNumberOfConsoleFonts
+@ stdcall GetNumberOfConsoleFonts()
 @ stdcall GetNumberOfConsoleInputEvents(long ptr)
 @ stdcall GetNumberOfConsoleMouseButtons(ptr)
 @ stdcall GetOEMCP()
-- 
2.6.1

